_targets/build.project.armv7m7-imxrt106x

```diff
@@ -67,7 +119,9 @@ b_build_target() {
 	b_log "Building plo"
 	make -C plo clean all
 
-	b_mkscript
+	MAGIC_USER_SCRIPT=$((($RANDOM % 0xffff) | (($RANDOM % 0xffff) << 16)))
+	b_mkscript_user
```

Comment:
I don't like that the scripts are implicitly removed with each recompilation by `make -C plo clean`. As they are created by the script, they should be "managed" by the script (so - explicitly removed by it when needed?)

Reply:
Compilation with clean is removed. Script checks whether file exists and removes it or not.